coral/README.md

```diff
@@ -0,0 +1,87 @@
+- `dev`: starts the frontend app for development
+- `lint`: runs a code format check and if no error is found, lints the code. 
+  - the linting script does not mutate your code. See [Linting and code formatting](#linting-and-code-formatting) for more info.
+- `preview`: builds a preview production build _locally_
+- `reformat`: runs the code formatter (prettier) and reformat all code
+- `test-dev`: runs all test tests related to changed files in a watch mode
+- `test`: runs all tests one time
+- `typecheck`: runs the TypeScript compiler (tsc)
+
+## Tech stack
+
+### App development
+- TypeScript - 📃 [documentation](https://www.typescriptlang.org/) | 🐙 [repository](https://github.com/microsoft/TypeScript)
+- React - 📃 [documentation](https://reactjs.org/docs/getting-started.html) | 🐙 [repository](https://github.com/facebook/react/)
+- Vite - 📃 [documentation](https://vitejs.dev/guide/) | 🐙 [repository](https://github.com/vitejs/vite)
+
+### Testing
+- Jest - 📃 [documentation](https://jestjs.io/docs/getting-started) | 🐙 [repository](https://github.com/facebook/jest)
+- React Testing Library - 📃 [documentation](https://testing-library.com/docs/react-testing-library/intro/) | 🐙 [repository](https://github.com/testing-library/react-testing-library)
+
+📃 You can find more detailed information about testing in our docs for [Frontend Testing](docs/frontend-testing.md).
+
+### Linting and code formatting
+How we keep our app's codebase looking consistent and nice 💅🏼
+
+- [Prettier](https://prettier.io/) for code formatting
+- [ESlint](https://eslint.org/) and various plugins for linting
+
+The script `lint` runs a prettier check and eslint after. It does not mutate your code in any way. If you want to format your code or let eslint fix it for you, you can run:
+
+1. First: `pnpm reformat` (prettier formatting)
+2. After that: `pnpm eslint --fix` (eslint in fix mode)
+
+ℹ️ It's convenient to let prettier and eslint auto-format your code "on save" by your IDE or editor.
+
+## Styling 
+
+We use the component library of Aiven's design system:
+  - 📃 [documentation](https://aiven-ds.netlify.app/)
+  - the repository is open source, but `private` at the moment
+
+As a rule, please don't use css classes from the design system. All styles should be created by using the existing components and their properties. 
+
+__🔄 Work in progress related to styles__
+- We plan adding css variables based on the design system's tokens.
+- We plan having a custom theme for Klaw. This will be used instead of the Aiven theme.
+
+## Documentation
+
+We've a more detailed document about our thinking about [Docmentation](docs/documentation.md). 
+
+We added `alex` ( 📃 [documentation]( https://alexjs.com/) | 🐙 [repository](https://github.com/get-alex/alex)) as an **optional** linting tool. It checks text documents and highlights language that potentially could be insensitive or inconsiderate. It is not part of any automated process. 
```

Comment:
I would opt for reducing the number of dependencies for the project to minimal. Every dependency will introduce additional maintenance work and exposes the project to possible security concerns. Having a tool to check our tone of voice and the used terminology is not a must, but a nice to have feature.

I would suggest that we remove `alex` as dependency and  alter this documentation to describe how to run the command without installing it as project depedency (and node script).

E.g. `npx alex`

However, I am not quite sure how important the configuration is (`.alexrc`). Also, instead of having a separate cofiguration file, we could include:
```
  "alex": {
    "noBinary": true,
    ...
  },
```
in the `package.json`. Less files in the repo and if someone happens to run a tool we promote, it would use the preset config.

Reply:
Good point! And you're of course right, if it's not part of any automated process but an optional recommendation, we don't need to introduce it as dependency. Will update it! 

I did not see a way to run alex with e.g. a flag for certain configs, so we have to add our preferred config. I can move it in the `package.json` though! (it's really a small config :D ).